File: _01_Languages/CSharp/src/_01_basicos/_02_TiposDeDatos.cs
/*
 * 🧩 Enunciado:
 * Declara variables de diferentes tipos de datos y muéstralas por consola.
 */

using System;

class _02_TiposDeDatos
{
    static void Main()
    {
        int edad = 25;
        double altura = 1.80;
        bool estudiante = true;
        char inicial = 'C';
        string nombre = "Carlos";

        Console.WriteLine($"Nombre: {nombre}");
        Console.WriteLine($"Edad: {edad}");
        Console.WriteLine($"Altura: {altura} m");
        Console.WriteLine($"¿Es estudiante?: {estudiante}");
        Console.WriteLine($"Inicial: {inicial}");
    }
}

File: _01_Languages/Java/ejercicios/Soluciones/_02_TiposDeDatos.java
package _01_Languages.Java.ejercicios.Soluciones;

/*
Ejercicio 2: Tipos de Datos y Variables
Declara variables de distintos tipos (int, double, char, boolean, String)
y muestra sus valores por pantalla.
*/

public class _02_TiposDeDatos {
    public static void main(String[] args) {
        int edad = 25;
        double altura = 1.75;
        char inicial = 'A';
        boolean estudiante = true;
        String nombre = "Lucía";

        System.out.println("Nombre: " + nombre);
        System.out.println("Edad: " + edad);
        System.out.println("Altura: " + altura + " m");
        System.out.println("Inicial: " + inicial);
        System.out.println("¿Es estudiante?: " + estudiante);
    }
}
